shap/explainers/tf_utils.py

```diff
@@ -43,7 +43,7 @@ def _get_graph(explainer):
     if not tf.executing_eagerly():
         return explainer.session.graph
     else:
-        return explainer.model_output.graph
+        return getattr(explainer.model_output, "graph", None)
```

Comment:
Starting with I think tf2.3, this will not contain a reference to the graph, or any of its ops anymore